.gitignore

```diff
@@ -109,6 +109,7 @@ scripts/guid_blacklist.txt
 
 # Generated json
 website/static/js/_allLogTexts.json
+website/static/js/_anonymousLogTexts.json
```

Comment:
I agree, but wouldn't consider this a blocker. It would be a good thing to do in the clean-up PR to remove all the code from the old logging system.